### ISSUE TITLE
Linux, 1.6.2 Alpha, startup-messages/shutdown message

### ISSUE THREAD

**Issue by MarkBergman-cbica** [CBICA/CaPTk#365]
**Describe the bug**

```
[bergman@huajiao ~]$ ~/CaPTk/v1.6.2.Alpha/captk 
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
```
(repeated)
```
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
An OpenGL Core Profile was requested, but it is not supported on the current platform. Falling back to a non-Core profile. Note that this might cause rendering issues.
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: cHRM chunk does not match sRGB
QObject::connect: No such slot fMainWindow::ApplicationGeodesicTraining()
QObject::connect:  (sender name:   'actionGeodesicTrainingSegmentation')
QObject::connect: No such signal fPCAEstimator::RunPCAEstimation(const int, const std::string, const std::string)
QObject::connect:  (sender name:   'fPCAEstimator')
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
```
(repeated)
```
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (8 8 8 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 8 8 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 8 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 8)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
```
(repeated)
```
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
js: WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips.
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
```
(repeated)
```
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 1)
Supplied directory name wasn't found: /datil/home/bergman/.CaPTk/1.6.2.Alpha/tmp_15432824966424522204
```

**To Reproduce**

**CaPTk Version**
Linux: 1.6.2 Alpha, from appinstaller

**Comment by sarthakpati** [comment]
A possible solution: https://stackoverflow.com/a/55235823/1228757

Ref: https://github.com/CBICA/CaPTk/issues/518#issuecomment-493986471

**Comment by sarthakpati** [comment]
Possibly related to #519

**Comment by sarthakpati** [comment]
This is not reproducible on Ubuntu 18.04 (however, the icon is not present #520).

**Comment by PhucNgo1711** [comment]
These errors were resolved on my VM 16.04 by installing a bunch of packages listed [here](https://doc.qt.io/qt-5/linux-requirements.html)

However, #519 still persists

**Comment by MarkBergman-cbica** [comment]
Issue still present with 1.7.0RC

**Comment by sarthakpati** [comment]
> Issue still present with 1.7.0RC

This fix hasn't been pulled into the latest build, yet.

**Comment by sarthakpati** [comment]
Fixed with #523
